<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlanetInfo</class>
 <widget class="QDialog" name="PlanetInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>291</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>NAME: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="l_name">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>PARENT : </string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="l_parent">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>COLOR : </string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="l_color">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>MASS : </string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="l_mass">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>RADIUS : </string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="l_radius">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>POSITION : </string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="l_pos">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>VELOCITY : </string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="l_veloc">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;/*************************&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt; * PLANET INFORMATION  *&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt; *************************/&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_trace">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Trace</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_untrace">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Untrace</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_trace</sender>
   <signal>released()</signal>
   <receiver>PlanetInfo</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_untrace</sender>
   <signal>released()</signal>
   <receiver>PlanetInfo</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
